packages/nvlsm/nvlsm.spec

```diff
@@ -0,0 +1,92 @@
+%global nvlsm_major 2025
+%global nvlsm_minor 03
+%global nvlsm_patch 1
+%global nvlsm_ver %{nvlsm_major}.%{nvlsm_minor}.%{nvlsm_patch}
+%global __brp_check_rpaths %{nil}
+%if "%{?_cross_arch}" == "aarch64"
+%global fm_arch sbsa
+%else
+%global fm_arch %{_cross_arch}
+%endif
+
+Name: %{_cross_os}nvlsm
+Version: %{nvlsm_ver}
+Release: 1%{?dist}
+Epoch: 1
+Summary: NVIDIA Subnet Manager component
+License: LicenseRef-NVIDIA-AWS-EULA
+URL: http://nvidia.com
+
+# NVIDIA rpms from 0 to 199
+Source0: https://developer.download.nvidia.com/compute/cuda/repos/rhel9/x86_64/nvlsm-%{nvlsm_ver}-1.x86_64.rpm
+Source1: https://developer.download.nvidia.com/compute/cuda/repos/rhel9/sbsa/nvlsm-%{nvlsm_ver}-1.aarch64.rpm
+Source2: NVidiaEULAforAWS.pdf
```

Comment:
This is not an NVIDIA rpm and it's in the sources from 0 to 199.

Reply:
This convention is consistent with the kmod packages, I can change it but we should change it for the others as well.